Libplanet/Net/Swarm.cs

```diff
@@ -2150,12 +2150,24 @@ private async Task ProcessFillTxs(CancellationToken cancellationToken)
                         $"Some tasks faulted during {nameof(GetTxsAsync)}().");
                 }
 
-                foreach (var task in tasks.Where(task => !task.IsFaulted))
+                foreach (Task<List<Transaction<T>>> task in tasks.Where(task => !task.IsFaulted))
```

Comment:
How about using an `if (task.IsFaulted) continue;` clause inside instead of `.Where()` method here?

Reply:
It's good. but I also realize that this change isn't the scope of this PR 😅 I'll divide commit.